include/star/sstar_to_dyn.h

```diff
@@ -27,9 +27,14 @@
  */
 
 void  addstar(dyn*,
-	      real t_rel=0,
+	      real t_cur=0,
```

Comment:
Renamed `t_rel` to `t_cur` to more closely match the function's name in add_star.C and to prevent clashing with the "actual" `t_rel`.